### PR TITLE
FEATURE: Support node-uris in ``neos:link.node`` and ``neos.uri.node`` ViewHelper

### PR DESCRIPTION
**Pull request by mficzel**
The ``neos:link.node`` and ``neos.uri.node`` ViewHelpers already support the nodePath as alternative to the node-object this change adds support for the ``node://``-uris of neos.

### CODE REVIEW

**Comment by mficzel** [comment]
This is an extract of the main feature of https://github.com/neos/neos-development-collection/pull/80 from @LarsNieuwenhuizen

File: TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Link/NodeViewHelper.php [hunk 1]
```diff
@@ -129,7 +129,7 @@ public function initializeArguments()
     /**
      * Renders the link. Renders the linked node's label if there's no child content.
      *
-     * @param mixed $node A node object or a string node path or NULL to resolve the current document node
+     * @param mixed $node A node object or a string node path(absolute or relative) a string node://-uri or NULL to resolve the current document node
```
Comment (skurfuerst) [inline TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Link/NodeViewHelper.php]: <nitpick>Could we adjust the text a little to make it better-understandable? Like "A node object; or a string node path (absolute or relative); or a string node://-uri; or NULL ....</nitpick>

Comment (skurfuerst) [inline TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Link/NodeViewHelper.php]: ADDITIONALLY: The Doc Comment up there in the class must be updated; as this is used to generate the RTD Docs.

**Comment by mficzel** [comment]
@skurfuerst i think i solved your nitpicks

**Pull request merged**